src/index.js

```diff
@@ -0,0 +1,41 @@
+import './style.css';
+import refresh from './assets/refresh.png';
+import enter from './assets/enter.png';
+
+const refreshImg = document.getElementById('refreshImg');
+const enterImg = document.getElementById('enterImg');
+
+refreshImg.src = refresh;
+enterImg.src = enter;
+
+const list = document.getElementById('list');
+const tasks = [
+  {
+    index: 0,
+    description: 'Wash the dishes',
+    completed: false,
+  },
+  {
+    index: 1,
+    description: 'Complete To Do list project',
+    completed: false,
+  },
+];
+
+const addTask = (task) => {
+  list.innerHTML += `
+    <li class="task">
+      <div class="task-info">
+        <input type="checkbox">
+        <p>${task.description}</p>
+      </div>
```

Comment:
- [ ] Nice one dynamically rendering the tasks, but kindly note that the `checkbox` or in your case the  `checked` property of the checkbox  should be rendered dynamically also with respect to the `completed` in each objects, meaning it should be checked on `completed: true` and unchecked on completed false...

You could do something like. `<input class='checkbox' type="checkbox" ${task.completed && 'checked'}/>` or more clearer with ternary operator `<input class='checkbox' type="checkbox" ${elem.completed? 'checked' : ''}/>`

the `checked` property is like the `required` property, but what it does is that it triggers the checked icon of the checkbox, when it is present.